examples/Learning.ipynb

```diff
@@ -0,0 +1,693 @@
+        "It's important to boost our calculation speed so use the GPU"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 4,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "3VXUKMUa8tZz",
+        "outputId": "0b54b4f9-8d14-489f-f6ec-04944412e470"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "cuda\n"
+          ]
+        }
+      ],
+      "source": [
+        "device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')\n",
+        "print(device)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Data preparation"
+      ],
+      "metadata": {
+        "id": "kkg5WEo9_3X5"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sP59rmCAkMET"
+      },
+      "source": [
+        "We are going to use non-aggregated data. So, read the *train*, *val* and *test* data from csv-files"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "L8H63lBr5lJe"
+      },
+      "outputs": [],
+      "source": [
+        "train_data = pd.read_csv(\"fixed_train_crowd_alpha047 (1).csv\")\n",
+        "val_data = pd.read_csv(\"fixed_val_clean_alpha047.csv\")\n",
+        "test_data = pd.read_csv(\"fixed_test.csv\")\n",
+        "pd.options.mode.chained_assignment = None"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dpIa7B22kkrY"
+      },
+      "source": [
+        "Preparing our data: choose the necessary columns and change labels from '*pos*'-'*neg*' to '*1*'-'0'"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "metadata": {
+        "id": "G6plSCtN5lMz"
+      },
+      "outputs": [],
+      "source": [
+        "train_data = train_data[['INPUT:text', 'OUTPUT:result', 'ASSIGNMENT:worker_id']]\n",
+        "train_data['OUTPUT:result'] = train_data['OUTPUT:result'].replace('pos', int(1))\n",
+        "train_data['OUTPUT:result'] = train_data['OUTPUT:result'].replace('neg', int(0))\n",
+        "\n",
+        "val_data = val_data[[\"text\", \"label\"]]\n",
+        "val_data['label'] = val_data['label'].replace('pos', int(1))\n",
+        "val_data['label'] = val_data['label'].replace('neg', int(0))\n",
+        "\n",
+        "test_data = test_data[['text', 'label']]\n",
+        "test_data['label'] = test_data['label'].replace('pos', int(1))\n",
+        "test_data['label'] = test_data['label'].replace('neg', int(0))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-OKEjdb3l1KB"
+      },
+      "source": [
+        "Since the ***id_worker*** field has the string type, we need to convert it to the int type. To do this, we will assign a unique string values to all unique int values in the field"
```

Comment:
Please replace `***` with `**` everywhere. It shows an extra star instead of making the text bold.